manifests/init.pp

```diff
@@ -67,28 +75,30 @@
     ensure => 'installed',
   }
 
-  if $manage_init_script == true {
+  # Don't manage init script for Ubuntu
+  if ($manage_init_script == true) and ($::osfamily != 'Ubuntu') {
```

Comment:
This would silently ignore when ```$mange_init_script``` is set to ```true``` for Ubuntu. Instead we should use ```fail()``` with a message that the usage of ```$manage_init_script``` is not supported on Ubuntu.

Pseudo code:
```
if $manage_init_script == true {
  if $::operatingsystem == 'Ubuntu' {
    fail ('manage_init_script is not supported on Ubuntu')
  }
...
```